core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -90,37 +90,46 @@ object ConsumerGroupCommand {
 
     protected def opts: ConsumerGroupCommandOptions
 
-    protected def getLogEndOffset(topic: String, partition: Int): LogEndOffsetResult
+    protected def getLogEndOffset(topic: Option[String], partition: Option[Int]): LogEndOffsetResult
 
     protected def describeGroup(group: String): Unit
 
-    protected def describeTopicPartition(group: String,
+    protected def describeMemberTopicPartition(group: String,
+                                         topic: String,
                                          topicPartitions: Seq[TopicAndPartition],
                                          getPartitionOffset: TopicAndPartition => Option[Long],
-                                         getOwner: TopicAndPartition => Option[String]): Unit = {
-      topicPartitions
-        .sortBy { case topicPartition => topicPartition.partition }
-        .foreach { topicPartition =>
-          describePartition(group, topicPartition.topic, topicPartition.partition, getPartitionOffset(topicPartition),
-            getOwner(topicPartition))
-        }
+                                         getMemberId: TopicAndPartition => Option[String],
+                                         getOwner: TopicAndPartition => Option[String],
+                                         getOwnerHost: TopicAndPartition => Option[String]): Unit = {
+      if (topicPartitions.isEmpty)
+        describePartition(group, None, None, None, getMemberId(null), getOwner(null), getOwnerHost(null))
+      else
+        topicPartitions
+          .sortBy { case topicPartition => topicPartition.partition }
+          .foreach { topicPartition =>
+            describePartition(group, Some(topicPartition.topic), Some(topicPartition.partition), getPartitionOffset(topicPartition),
+              getMemberId(topicPartition), getOwner(topicPartition), getOwnerHost(topicPartition))
+          }
     }
 
     protected def printDescribeHeader() {
-      println("%-30s %-30s %-10s %-15s %-15s %-15s %s".format("GROUP", "TOPIC", "PARTITION", "CURRENT-OFFSET", "LOG-END-OFFSET", "LAG", "OWNER"))
+      println("%-30s %-30s %-10s %-15s %-15s %-10s %-50s %-30s %s".format("GROUP", "TOPIC", "PARTITION", "CURRENT-OFFSET", "LOG-END-OFFSET", "LAG", "MEMBER-ID", "OWNER", "OWNER-HOST"))
     }
 
     private def describePartition(group: String,
-                                  topic: String,
-                                  partition: Int,
+                                  topic: Option[String],
+                                  partition: Option[Int],
                                   offsetOpt: Option[Long],
-                                  ownerOpt: Option[String]) {
+                                  memberOpt: Option[String],
+                                  ownerOpt: Option[String],
+                                  ownerHostOpt: Option[String]) {
```

Comment:
Could we change ownerOpt and ownerHostOpt to clientIdOpt and memberHostOpt, respectively?